indico/modules/categories/forms.py

```diff
@@ -117,3 +118,22 @@ class CreateCategoryForm(IndicoForm):
 
     title = StringField(_("Title"), [DataRequired()])
     description = IndicoMarkdownField(_("Description"))
+
+
+class SplitCategoryForm(IndicoForm):
+    first_category = StringField(_('Category name #1'), [DataRequired()],
+                                 description=_('Selected events will be put into a new sub-category with this title.'))
+    second_category = StringField(_('Category name #2'), [DataRequired()],
+                                  description=_('Events that were not selected will be put into a new sub-category '
+                                                'with this title.'))
+    event_id = HiddenFieldList(validators=[DataRequired()])
+    submitted = HiddenField()
+
+    def __init__(self, *args, **kwargs):
+        super(SplitCategoryForm, self).__init__(*args, **kwargs)
+        if kwargs.get('move_all', False):
+            self.first_category.label.text = _('Name for the new subcategory')
```

Comment:
Isn't that a bit long? This looks more like a description to me. Why not "Category name"?